visualizer/ui.js

```diff
@@ -394,12 +394,12 @@ class Ui extends events.EventEmitter {
 
   getDescriptionFromKey (key) {
     const keysToDescriptions = {
-      'core': `JS functions in core Node.js APIs. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#core">More info</a>`,
+      'core': `JS functions in core Node.js APIs.`,
       'all-v8': 'The JavaScript engine used by default in Node.js',
```

Comment:
Can we add a link here to the main V8 section heading (currently https://clinicjs.org/documentation/flame/09-advanced-controls/#v8 )